applications/ParticleMechanicsApplication/python_scripts/mpm_gid_output_process.py

```diff
@@ -0,0 +1,238 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications and dependencies
+import KratosMultiphysics.SolidMechanicsApplication as KratosSolid
+import KratosMultiphysics.ParticleMechanicsApplication as KratosParticle
+
+# Import time library
+import time
+
+# Check that KratosMultiphysics was imported in the main script
+KratosMultiphysics.CheckForPreviousImport()
+
```

Comment:
Try to reduce code duplication calling the gid_output_process in the core

Reply:
Basically, this output process is printing the particle information and results. So I cant reuse the currently available gid_output_process. I have tried that, and didn't work.

Reply:
Well, leave like this, we will check in another PR